packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1741,18 +1782,21 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   closeLocalShare() {
-    return Media.stopTracks(this.mediaProperties.shareTrack).then(() => {
-      Trigger.trigger(
-        this,
-        {
-          file: 'meeting/index',
-          function: 'closeLocalShare'
-        },
-        EVENT_TRIGGERS.MEDIA_STOPPED, {
-          type: EVENT_TYPES.LOCAL_SHARE
+    return Media.stopTracks(this.mediaProperties.shareTrack)
```

Comment:
Is there a reason to not use `stopStream`?

Reply:
Yes because it's a single track and not a stream. So `.stopStream()` calls `.getTracks()` on the stream to get the individual tracks but this is already a track itself.